get-to-know-syskit-point/generate-reports/licenses-reports.md

```diff
@@ -1,114 +1,157 @@
 # Licenses Reports
 
-Reports in this section help manage your Microsoft 365 licenses. Use them to discover assigned, unassigned or inactive Microsoft 365 licenses, check their overall cost. You can also use these reports to identify underutilized licenses to drive adoption and reduce the cost of the licenses by assigning less expensive ones to users.
+Reports in this section **help manage your Microsoft 365 
+licenses**. 
+
+In this article you can locate information on the following license reports:
+* [Licenses Overview Report](#licenses-overview-report)
+* [Licenses Usage by Service Report](#licenses-usage-by-service-report)
+* [License Distribution](#license-distribution)
+* [Inactive Licenses Report](#inactive-licenses-report)
+
+Use them to **discover assigned, unassigned or inactive Microsoft 365 licenses** and check their overall cost. 
```

Comment:
ovdje bi malo promijenio wording. 
1) ...'unassigned or Microsoft 365 licenses assigned to inactive users'...
2) moramo biti oprezni i ne spominjati reduction troškova nego 'optimize license cost'. MS nam je partner pa ne možemo raditi kontra njihove zarade, recimo to tako.